### PR TITLE
Tanya/migration

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
## Proposed changes

1) fixes in useMigrate
2) removed unused lowBalance from useMasterBalances and useServiceBalances
3) fixed most of the migration related components, will complete and test once app is more stable

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by truemiller — APPROVED** [review]

File: frontend/components/ManageStakingPage/StakingContractSection/CantMigrateAlert.tsx [hunk 1]
```diff
@@ -20,33 +27,65 @@ const { Text } = Typography;
...
 
-  const totalOlasRequiredForStaking = getMinimumStakedAmountRequired(
-    serviceTemplate,
-    stakingProgramId,
-  );
+  // should find in STAKING_PROGRAMS based on stakingProgramIdToMigrateTo?
+  const chainIdToMigrateTo = ChainId.Gnosis;
+
+  const requiredStakedOlas =
+    service &&
```
Comment (Tanya-atatakai) [inline frontend/components/ManageStakingPage/StakingContractSection/CantMigrateAlert.tsx:57]: copy-pasted this from AgentNotRunningButton but wonder should we check service

File: frontend/components/ManageStakingPage/StakingContractSection/MigrateButton.tsx [hunk 2]
```diff
@@ -111,20 +120,20 @@ export const MigrateButton = ({
...
             });
 
             // start service after updating
-            await ServicesService.startService(
-              serviceTemplate.service_config_id,
-            );
+            await ServicesService.startService(serviceConfigId);
 
-            await updateStakingProgram(); // TODO: refactor to support single staking program & multi staking programs, this on longer works
+            // await updateStakingProgram(); // TODO: refactor to support single staking program & multi staking programs, this on longer works
```
Comment (Tanya-atatakai) [inline frontend/components/ManageStakingPage/StakingContractSection/MigrateButton.tsx:136]: unsure what this should do, would appreciate any advice

File: frontend/components/ManageStakingPage/StakingContractSection/CantMigrateAlert.tsx (hunk 1 above)
Comment (truemiller) [reply frontend/components/ManageStakingPage/StakingContractSection/CantMigrateAlert.tsx:57]: can check this without service condition

File: frontend/components/ManageStakingPage/StakingContractSection/MigrateButton.tsx (hunk 2 above)
Comment (truemiller) [reply frontend/components/ManageStakingPage/StakingContractSection/MigrateButton.tsx:136]: was previously a function refetch staking program data

**Pull request merged**